test/data/CEDDL.ts

```diff
@@ -23,8 +25,10 @@ export interface CEDDL {
 /**
  * Because of the wide range of methods for categorization, an object literal is provided for categories.
  */
-export interface Category {
+export interface PageCategory {
```

Comment:
I had noticed that there was some overlap in the *Category models and had created a generic `Category` object.  But I think what you've done is encapsulate those shared properties in each domain category - presumably because changing them would not depend on updating the super interface.  I think that this is the right approach.